drivers/flash/CMakeLists.txt

```diff
@@ -56,6 +56,12 @@ if(CONFIG_SOC_FLASH_STM32)
   endif()
 endif()
 
+if (CONFIG_BUILD_WITH_TFM)
+	zephyr_library_include_directories(
+      $<TARGET_PROPERTY:tfm,TFM_BINARY_DIR>/install/interface/include
```

Comment:
@SebastianBoe would it be possible for TF-M to provide an interface library that can export include folders, so that a library may simply link with TF-M interface lib, and not having to know about internal `TFM_BINARY_DIR` as well as `install/interface/include` path ?

Reply:
Done: https://github.com/zephyrproject-rtos/zephyr/pull/42733

Not sure what to do about this PR. rebase it onto 42733 perhaps?

Reply:
will do